middlewares/validators/validator.helper.js

```diff
@@ -390,6 +390,31 @@ function searchValidator (fieldLocation, fieldname) {
     });
 }
 
+function searchSortValidator (fieldLocation, fieldName){
+    const searchSort = setProperValidationChainBuilder(fieldLocation, fieldName, "Invalid sort criteria")
+    return searchSort.optional({checkFalsy: true})
+        .custom((value) => {
+            let modelString = param("model", "Corresponding model not found");
+            let model;
+            if(modelString.equals('hacker')){
```

Comment:
Does this mean we're only able to search through hackers? Will there be functionality for search through other models in the future?

Reply:
Yeah, I'm working on that next, this is necessary so the fields that are searchable will also be sortable